src/logic.js

```diff
@@ -0,0 +1,79 @@
+export function Task(title, description, dueDate, priority) {
+  this.title = title;
+  this.description = description;
+  this.dueDate = dueDate;
+  this.priority = priority;
+  this.complete = false;
+}
```

Comment:
__[Optional]__ This is a good place to use the ES6 `class` syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/))